README.md

```diff
@@ -124,6 +119,21 @@ rm /etc/yum.repos.d/balena-etcher.repo
 rm /etc/yum.repos.d/balena-etcher-source.repo
 ```
 
+#### OpenSUSE LEAP & Tumbleweed install (zypper)
+
+```sh
+curl -1sLf \
+  'https://dl.cloudsmith.io/public/balena/etcher/setup.rpm.sh' \
+  | sudo -E bash
```

Comment:
Reading through the install-instructions for the other OSes, it seems that this `curl ... | bash` line just adds the CloudSmith repositories to the system repositories, and then there's some other package-commands (`apt-get` / `dnf` / `yum`) to actually install the `balena-etcher-electron` package? So perhaps OpenSUSE is similar, and that also needs an additional package-command (`zypper`?) to actually install balenaEtcher?  (I can't be sure, as I've never used OpenSUSE myself)

Reply:
thanks, quite crowded part, i did not even notice it is missing (and outdated)